tests/karate/src/test/java/fe/net/mockbackend.feature

```diff
@@ -0,0 +1,20 @@
+@ignore @report=false
+Feature: Mock Backend
+
+  Scenario: Setup Mock-backend
+    * def getBackend =
+            """
+              function() {
+                var Backend = Java.type("fe.net.MockBackend")
+                return new Backend(9005)
+              }
+            """
+
+    * def backend = call getBackend
+    * karate.log('Backend started at ', backend.getPort())
+
+    * def stopBackend = function() { backend.stop() }
+
+    # Shutdown backend automatically after the end of a scenario and  feature
+    * configure afterScenario = stopBackend
+    * configure afterFeature = stopBackend
```

Comment:
the configuration of after scenario/feature should be done in the Background don't you think so ?